plugin-server/src/capabilities.ts

```diff
@@ -18,9 +18,25 @@ export function getPluginServerCapabilities(config: PluginsServerConfig): Plugin
             return { ingestion: true, ...sharedCapabilities }
         case 'async':
             return {
-                pluginScheduledTasks: true,
                 processPluginJobs: true,
                 processAsyncHandlers: true,
+                pluginScheduledTasks: true,
+                ...sharedCapabilities,
+            }
+        case 'async-handlers':
```

Comment:
non-blocking: maybe we call this exports now?

we were calling the grouping of jobs, exports, and scheduled tasks "async handlers"

Reply:
Ah yes ok I'll update 👍